packages/driver/package.json

```diff
@@ -66,6 +66,12 @@
     "zone.js": "0.9.0"
   },
   "files": [
-    "lib"
-  ]
+    "lib",
+    "patches"
+  ],
+  "workspaces": {
+    "nohoist": [
+      "*"
```

Comment:
Does this mean that the root will now no longer hoist these driver packages while the building of the binary will keep them in the driver package? (Referencing https://github.com/cypress-io/cypress/pull/6583/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L221-L223 in addition to this addition).

Reply:
previously, `yarn install` from root and `yarn install` from packages/driver would "nohoist" these packages, but `yarn workspace driver install` would not. now, all 3 of those commands will have the same `nohoist` behavior.

Reply:
actually, i think the behavior is exactly the same as before. i made this change just in the interest of keeping package-specific config per-package as much as possible.